src/components/ChatView.vue

```diff
@@ -105,7 +105,7 @@ export default {
 
 	data() {
 		return {
-			isChatScrolledToBottom: true,
+			isChatScrolledToBottom: false,
```

Comment:
Initial loading:
![image](https://github.com/user-attachments/assets/5c05562e-91e0-4853-86ab-7c6d9aecd7ff)

Let's hide, if messages aren't initialized yet? or keep as `true`? it will be true anyway in most cases after chat loaded, what are exceptions?

Reply:
Yeah, in most cases it is scrolled to bottom except for message hash in link or unread messages above the bottom view. Will set it to true again.